perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -1325,6 +1328,24 @@ sub waste_munge_request_form_fields {
     );
 }
 
+=head2 request_cost
+
+Calculate how much, if anything, a request for a container should be.
+
+=cut
+
+sub request_cost {
+    my ($self, $id) = @_;
+    my $cost;
+    $cost = $self->_get_cost('request_cost_blue_bin') if $id == 6; # $CONTAINER_BLUE_BIN
```

Comment:
Is it possible to have these numbers (6, 11) as constants?

Reply:
Yes, looks like I meant to do that from the comments (those constants are from a different file), will do